launch.py

```diff
@@ -158,7 +210,10 @@ def run_extensions_installers(settings_file):
 
 
 def prepare_environment():
-    torch_command = os.environ.get('TORCH_COMMAND', "pip install torch==1.12.1+cu113 torchvision==0.13.1+cu113 --extra-index-url https://download.pytorch.org/whl/cu113")
+    if gpu == "AMD" and os_name !="nt":
+        torch_command = os.environ.get('TORCH_COMMAND', "pip install torch torchvision --extra-index-url https://download.pytorch.org/whl/rocm5.2")
+    else:

```

Comment:
This will install cuda if `gpu == "AMD" and os_name == "nt"`

Reply:
This is expected as for now there's no torch implementation for amd working on windows

Reply:
When torch directml will be stable enough to work the code can be easily updated to include support for it. For now the pr just improves linux compatibility and adds os detection